lib/tts.rb

```diff
@@ -3,9 +3,9 @@
 require 'uri'
 
 module Tts
-  @@default_url = "http://translate.google.com/translate_tts"
+  @@default_url = "http://soundoftext.com/static/sounds/en/"
```

Comment:
@@default_url = "http://soundoftext.com/static/sounds/en/" should be

@@default_url = "http://soundoftext.com/static/sounds"